octopoes/octopoes/models/ooi/dns/records.py

```diff
@@ -185,3 +185,43 @@ class DNSCAARecord(DNSRecord):
     # without interior spaces or (2) a quoted string.
     value: str
     _natural_key_attrs = ["hostname", "flags", "tag", "value"]
+
+
+class DNSGPOSRecord(DNSRecord):
+    # RFC 1712
+    object_type: Literal["DNSGPOSRecord"] = "DNSGPOSRecord"
+    dns_record_type: Literal["GPOS"] = "GPOS"
+
+    latitude: str | None = None
+    longitude: str | None = None
+    altitude: str | None = None
+
+    @property
+    def natural_key(self) -> str:
+        sha = hashlib.sha1(self.value.encode("UTF-8")).hexdigest()
```

Comment:
utf8 has always been the default argument, but it also doesn't add anything since the encoding doesn't matter and the value should be ASCII (I think) anyways

```suggestion
        sha = hashlib.sha1(self.value.encode()).hexdigest()
```

Reply:
The remarks in this class also applies for the `DNSLOCRecord` class below